core/trino-main/src/main/java/io/trino/execution/Input.java

```diff
@@ -72,6 +68,12 @@ public String getCatalogName()
         return catalogName;
     }
 
+    @JsonProperty
+    public Object getCatalogVersion()
```

Comment:
Sorry, I should have been more clear, my question from earlier was whether we could use `CatalogVersion` as the type, rather than just `String`.